src/components/FormElements/DeclinationInput.vue

```diff
@@ -27,10 +29,10 @@
 
 <script>
 export default {
-  props: ['value', 'size'],
+  props: ['value', 'size', 'disabled'],
   data () {
     return {
-      localDec: null,
+      localDec: this.value,
```

Comment:
Here we default the localDec value to the prop that was passed in - I was having an issue when I was re-rendering the component, the dec wasn't displaying even though the value was being passed. So set the default to the value passed in.